app/2.0/start/toolbox/deploy.md

```diff
@@ -191,3 +191,8 @@ guide](https://www.firebase.com/docs/hosting/quickstart.html).
    The URL to your live site is listed in the output. You can also open
    the site in your default browser by running `firebase open hosting:site`.
 
+## Next Steps
+
+Ready for what's next? The App Toolbox comes with a built-in service worker - a script that
+enables serving pages from the browser cache to provide a better experience in offline and 
+spotty network situations. As a next step, try (adding a service worker)[/{{{polymer_version_dir}}}/toolbox/service-worker] to your app.
```

Comment:
Service workers are pretty advanced, easy to mess up, and we don't really have step-by-step instructions in that section. I would rather direct them to other docs (learn more about Polymer, try the element tutorial if they haven't already, maybe direct them to the app toolbox landing page to learn about more parts of the app toolbox.